package.json

```diff
@@ -28,15 +28,17 @@
     "expo": "^49"
   },
   "devDependencies": {
-    "expo-module-scripts": "^3.0.11"
+    "expo-module-scripts": "^3.1.0"
   },
   "//": "Override vulnerable dependency until expo updates it.",
   "resolutions": {
+    "expo-module-scripts/@babel/core": "^7.23.2",
+    "expo-module-scripts/@babel/traverse": "^7.23.2",
```

Comment:
these are supposed to _both_ be 7.23.2?

Reply:
The `expo-module-scripts` has not been updated yet to use the recent version of these libraries, so that is the only way to workaround this limitation and upgrade the version of the libraries.

But yep, I have tested, and everything keeps working fine with these latest versions.